stacks_queues/stack.py

```diff
@@ -21,18 +21,21 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if self.store.empty():
+            raise StackEmptyException
+        else:
+            return self.store.remove_first()
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        return self.store.empty()
 
     def __str__(self):
         """ Returns the Stack in String form like:
             [3, 4, 7]
             Starting with the top of the Stack and
             ending with the bottom of the Stack.
         """
-        pass
+        return self.store.__str__()
```

Comment:
The `__str__` method is an example of what we call operator overloading. Operator overloading is used for methods and operators such as `str()`, `==`, and `+` where Python actually has a predefined way it will attempt to do these operations or methods on any object or data type. Methods like `__str__`, `__eq__`, and `__add__` allow us to either extend or overwrite the predefined functionality with our own desired functionality.

When we implement the function, we use the dunders around the function name (ex. `__str__` ) to indicate that we are about to do operator overloading. However, when we call the function, we can remove the dunders.
```suggestion
        return str(self.store)
```